### ISSUE TITLE
docComments don't work for code inside #if

### ISSUE THREAD

**Issue by dungi** [nicklockwood/SwiftFormat#1528]
Hey,

since the last update you fixed `docComments` inside if statements. Sadly this also affects codes inside `#if` blocks

like

```
#if DEBUG

/// return 3
func returnNumber() {
  return 3
}

#endif
```

Will be converted to

```
#if DEBUG

// return 3
func returnNumber() {
  return 3
}

#endif
```

**Comment by nicklockwood** [comment]
Thanks for flagging this, I'll get it fixed.

**Comment by nicklockwood** [comment]
@dungi fixed in 0.52.5
